docs/release-train-issue-template.md

```diff
@@ -0,0 +1,69 @@
+Ideally replacing variables could become a script you can run on your local machine.
+
+Variables to be expanded in this template:
+- $VERSION$=??? 
+
+-->
+
+### before the release
+
+- [ ] Create a new milestone for the [next version](https://github.com/akka/akka-persistence-cassandra/milestones)
+- [ ] Check [closed issues without a milestone](https://github.com/akka/akka-persistence-cassandra/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20no%3Amilestone) and either assign them the 'upcoming' release milestone or `invalid/not release-bound`
+- [ ] Make sure all important / big PRs have been merged by now
+
+### Preparing release notes in the documentation / announcement
+
+- [ ] For non-patch (minor/major) releases: rename the 'akka-persistence-cassandra-x.x-stable' and 'akka-persistence-cassandra-supported-x.x-stable' reporting projects in [WhiteSource](https://saas.whitesourcesoftware.com/Wss/WSS.html) accordingly (unfortunately this requires permissions that cannot be shared outside of Lightbend)
+- [ ] For non-patch releases: Create a news item draft PR on [akka.github.com](https://github.com/akka/akka.github.com), using the milestone
+- [ ] Move all [unclosed issues](https://github.com/akka/akka-persistence-cassandra/issues?q=is%3Aopen+is%3Aissue+milestone%3A$VERSION$) for this milestone to the next milestone
+- [ ] Close the [milestone](https://github.com/akka/akka-persistence-cassandra/milestones?direction=asc&sort=due_date)
+
+### Cutting the release
+
+- [ ] Make sure any running [Travis builds](https://travis-ci.org/akka/akka-persistence-cassandra) for the commit you would like to release have completed.
+- [ ] Create the [release notes](https://github.com/akka/akka-persistence-cassandra/releases) with the next tag version `v$VERSION$`, title, release description and contributors (the latter generated by [`sbt-authors`](https://github.com/2m/authors) (eg. `sbt authors v1.0.1 HEAD`))
+- [ ] Check that Travis CI release build has executed successfully (Travis will start a [CI build](https://travis-ci.org/akka/akka-persistence-cassandra/builds) for the new tag and publish artifacts to Bintray and documentation to Gustav)
+- [ ] Go to [Bintray](https://bintray.com/akka/maven/akka-persistence-cassandra) and select the just released version
+- [ ] Log in, go to the Maven Central tab, check the *Close and release repository when done* checkbox and sync with Sonatype (using your Sonatype TOKEN key and password)
+
+### Check availability
+
+- [ ] Check [reference](https://doc.akka.io/docs/akka-persistence-cassandra/$VERSION$/) documentation
+- [ ] Check the release on [Maven central](https://repo1.maven.org/maven2/com/typesafe/akka/akka-persistence-cassandra_2.12/$VERSION$/)
+
+### When everything is on maven central
+  - [ ] `ssh akkarepo@gustav.akka.io`
+    - [ ] update the `current` links on `repo.akka.io` by with the script:
+         ```
+         ./update-akka-persistence-cassandra-current-version.sh $VERSION$
```

Comment:
Update the shell script to change the `1.0` link as well.

Reply:
1.0 links to current or opposite